server/svix-server/src/cfg.rs

```diff
@@ -136,17 +136,17 @@ pub struct ConfigurationInner {
     pub redis_pool_max_size: u16,
 
     /// What kind of message queue to use. Supported: memory, redis (must have redis_dsn or
-    /// queue_dsn configured).
+    /// queue_dsn configured), and rediscluster (must also have a DSN set).
     pub queue_type: QueueType,
     /// The DSN for the Redis-backed queue. Overrides `redis_dsn`. (can be left empty if not using
     /// redis)
     pub queue_dsn: Option<String>,
 
-    /// What kind of cache to use. Supported: memory, redis (must have redis_dsn or cache_dsn
-    /// configured), none.
+    /// What kind of cache/persistent store to use. Supported: memory, redis (must have redis_dsn or
```

Comment:
I'll remove the `/persistent store` -> you can just just `cache/persistent store` to `persistent cache`.